games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+VIRTUALX_REQUIRED="test"
+
+inherit java-utils-2 virtualx
+
+DESCRIPTION="A Rom Manager entirely written in Java and released under GPL v3"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip"
```

Comment:
it has `tar.gz` sources. and use `-> ${P}.tar.gz` therefore

Reply:
Sources can't be compiled since it needs gradle 4.3 => prebuilt zip is used

Reply:
So bump gradle?

Reply:
too big for me

Reply:
Btw even older gradle-bin  versions available in portage are still masked